### PR TITLE
Add documentation for the Contentful Laravel integration.

### PR DESCRIPTION
**Pull request by realityking**

### CODE REVIEW

File: docs/php/index.md [hunk 1]
```diff
@@ -35,16 +35,27 @@ The Sync API allows you to keep a local copy of all content of a space up-to-dat
 How to setup the ContentfulBundle in your Symfony application is explained in this tutorial.<br>
 [View the tutorial](/developers/docs/php/tutorials/getting-started-with-contentful-and-symfony/)
 
+### Getting started with Contentful and Laravel
+How to setup the Laravel integration in your application is what you'll lean in this tutorial.<br>
```
Comment (neonichu) [inline docs/php/index.md]: > learn

File: docs/php/index.md [hunk 2]
```diff
@@ -35,16 +35,27 @@ The Sync API allows you to keep a local copy of all content of a space up-to-dat
...
+Simple integration to make it easier to use the Content Delivery API in a Laravel app<br>
+[View on GitHub](https://github.com/contentful/contentful-laravel)<br>
+
 ## Apps
 
 ### Space template apps
 These apps are the best way to start building your platform with Contentful. They show you how to start your project based on example templates available in the Contentful web app.
 
-Generic product catalogue / e-commerce app connected with an example space in Contentful.<br>
+Generic product catalogue / e-commerce app connected with an example space in Contentful, using Symfont.<br>
```
Comment (neonichu) [inline docs/php/index.md]: > Symfony

File: docs/php/tutorials/getting-started-with-contentful-and-laravel.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+---
+page: :docsGettingStartedLaravel
+---
+This tutorial will show you how to setup the Content Delivery API in your Laravel application and how to access you content
```
Comment (neonichu) [inline docs/php/tutorials/getting-started-with-contentful-and-laravel.md]: > your content

File: docs/php/tutorials/getting-started-with-contentful-and-laravel.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+Before the Contentful SDK can be configured, the necessary config files have to be published. To do so execute the following command:
+
+~~~ bash
+php artisan vendor:publish
+~~~
+
+This will add a file called `contentful.php` to your `/config` folder.
+
+Now open that file and fill add your space ID and API key.
```
Comment (neonichu) [inline docs/php/tutorials/getting-started-with-contentful-and-laravel.md]: should be either "fill in" or "add"

File: docs/php/tutorials/getting-started-with-contentful-and-laravel.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+To discover how to use the Contentful client, check out the
+[getting started with Contentful and PHP](/developers/docs/php/tutorials/getting-started-with-contentful-and-php/) tutorial.
+
+## Conclusion
+
+Now you should be familiar with the basics of how to use Contentful in a Laravel application. You can find the integration on
+[GitHub](https://github.com/contentful/contentful-laravel/) and [Packagist](https://packagist.org/packages/contentful/laravel).
+To get a deeper understanding, read some of our other [PHP tutorials](/developers/docs/php/#tutorials). If you find a bug,
+or have an idea how to further integrate with Symfony, please open an [issue on GitHub](https://github.com/contentful/contentful-laravel/issues).
```
Comment (neonichu) [inline docs/php/tutorials/getting-started-with-contentful-and-laravel.md]: Symfony => Laravel, probably?

**Comment by realityking** [comment]
All comments addressed, could you look through the fix really quick? Thanks :)

**Comment by neonichu** [comment]
:+1:

File: docs/php/tutorials/getting-started-with-contentful-and-laravel.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+---
+This tutorial will show you how to setup the Content Delivery API in your Laravel application and how to access your content
+inside the framework.
+
+{: .note}
+The ContentfulBundle and the PHP SDK are currently in beta and their APIs may change before a stable release.
+
+## Requirements
+
+The Contentful Laravel integrations requires at least PHP 5.5.9 and Laravel 5. PHP 7 is are supported.
```
Comment (mariobodemann) [inline docs/php/tutorials/getting-started-with-contentful-and-laravel.md]: `are` is not needed here ...

File: docs/php/tutorials/getting-started-with-contentful-and-laravel.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+## Installation
+
+The easiest way to install the Laravel integration is is to use [Composer][1]. If you've downloaded
```
Comment (mariobodemann) [inline docs/php/tutorials/getting-started-with-contentful-and-laravel.md]: `sqrt( is is )` aka, there is an `is` to much ...

**Comment by mariobodemann** [comment]
Except for the small typos above, lgtm.

**Pull request merged**